single-nb_newsletter.php

```diff
@@ -2,7 +2,7 @@
 /**
  * The template for displaying Newsletters.
  *
- * @package WP_Newsletter_Builder
+ * @package wp-newsletter-builder
```

Comment:
What is the reason for this change? Is it to use the namespace of the plugin name rather than the PHP namespace?